server/prisma/generator/factories/user.factory.ts

```diff
@@ -21,13 +22,21 @@ const createUsers = async (
   };
   const owner = await prisma.users.create({ data: ownerData });
 
-  const adminData: Prisma.usersCreateInput = {
+  const chapter1AdminData: Prisma.usersCreateInput = {
     email: 'admin@of.a.chapter',
```

Comment:
Since we have `admin@of.chapter.two` how about renaming this to `admin@of.chapter.one`? I think that's a little nicer.

Reply:
That's fine with me!